src/pyHepGrid/src/header.py

```diff
@@ -41,17 +41,13 @@
 # This should not be changed unless you really know what you are doing!
 # Grid config
 
-gsiftp = "gsiftp://se01.dur.scotgrid.ac.uk/dpm/dur.scotgrid.ac.uk/home/" + \
-    "pheno/generated/"
-LFC_HOST = "lfc01.dur.scotgrid.ac.uk"
-LFC_CATALOG_TYPE = "lfc"
-runfile = "nnlorun.py"
-runmode = "NNLOJET"
 sandbox_dir = "test_sandbox"
 arc_direct = True
 split_dur_ce = True
 slurm_kill_exe = "{0}/kill_server.py".format(
     os.path.dirname(os.path.realpath(__file__)))
+ce_listfile = "{0}/computing_elements.txt".format(
+    os.path.join(os.path.dirname(__file__), "../extras/"))
```

Comment:
Is there a good reason why we keep these in a file rather than in a list in our headers?

Reply:
Historical? Maybe the idea was that you can have you own list which is easy to update. Feel free to change this, but the complete "find most free cores" logic is broken anyway.

Reply:
When @DWalker487 discovered that we could use other CE we didn't know which ones we could use or even if we were really allowed to do so. So we had a list that we were updating and commenting as we discovered that this or that CE was broken, or very slow, or never have enough cores etc.